benchmarks/benchmark_threshold_sauvola.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+from skimage import filters
+
+class ThresholdSuite:
+    """Benchmark for transform routines in scikit-image."""
+
+
+    def setup(self):
+        self.image = np.zeros((2000, 2000))
+        self.image3D = np.zeros((300, 300, 30))
+
+        idx = np.arange(500, 700)
+        idx3D = np.arange(100, 100)
+
+        self.image[idx[::-1], idx] = 255
+        self.image[idx, idx] = 255
+
+        self.image3D[idx3D[::-1], idx3D, idx3D] = 255
+        self.image3D[idx3D, idx3D, idx3D] = 255
```

Comment:
If you are going to use 255 as the value, make sure that `np.zeros` is called with `dtype=np.uint8`. Otherwise, use 1.0 here. See https://scikit-image.org/docs/dev/user_guide/data_types.html for details.